eos-tech-support/eos-fix-mbr-swap-removal

```diff
@@ -44,22 +48,43 @@ exit_error() {
     exit ${2}
 }
 
-root_disk=${1}
-sfdisk -d ${root_disk} > ${PARTITIONS_FILE}
-
-# Check this is a MBR partiton table
-label_type=$(sed -n -e 's/^label: [ ]*\(.*\)$/\1/p' ${PARTITIONS_FILE})
-if [[ ${label_type} != "dos" ]] ; then
-    exit_error "Error: This is not a MBR partition table." 2
+if [[ ${#} -eq 1 ]] ; then
+    disks=${1}
+    print_log "Using disk specified by user: ${disks}."
+else
+    disks=$(echo $(lsblk --nodeps --noheadings --list --output NAME  | sed -e 's:\(.*\):/dev/\1:'))
```

Comment:
$(echo seems redundant, can you just do disks=$(lsblk ....

Reply:
I've added `echo` to get a list instead multiple lines from `lsblk`.